test/validator/samples/lets-on-component/warnings.json

```diff
@@ -0,0 +1,26 @@
+[
+	{
+		"code": "missing-declaration",
+		"message": "'count' is not defined",
+		"start": {
+			"column": 22,
+			"line": 10
+		},
+		"end": {
+			"column": 27,
+			"line": 10
+		}
+	},
+	{
+		"code": "missing-declaration",
+		"message": "'count' is not defined",
```

Comment:
is it possible to enhance the message in this case? Like "`count` is defined on the default slot but used in a named slot where it's not available"

Reply:
I was thinking of something similar but as a result of the changes there's no way to get that information now as the inline component doesn't happen to be a parent scope of where it's used.

I tried to hack around by finding the nearest InlineComponent and looking through its attributes but that isn't all that clean.

Reply:
Added some rudimentary tracking for it, seems to be a bit bugged because `let:` destructurings return an ObjectExpression instead of an ObjectPattern. Marked it as a TODO for now.